config.dev.yaml

```diff
@@ -61,6 +61,8 @@ services:
       #   - x-request-id
       # Uncomment to restrict connections to localhost only
       # interface: localhost
+      # Uncomment to disable the X-Powered-By header
```

Comment:
Uncomment to "disable" or to "enable", I'm confused?

Reply:
I removed the flag support by accident when cleaning up the git history. It's back in `app.js` now: https://github.com/kartotherian/kartotherian/pull/66/files#diff-0364f57fbff2fabbe941ed20c328ef1aR102